plugins/ssr/ssrvernac.ml4

```diff
@@ -546,7 +546,7 @@ GEXTEND Gram
           d = G_vernac.def_body ->
           let s = coerce_reference_to_id qid in
 	  Vernacexpr.VernacDefinition
-	    ((Some Decl_kinds.Global,Decl_kinds.CanonicalStructure),
+	    ((NoDischarge,Decl_kinds.CanonicalStructure),
```

Comment:
Isn't this a change of semantics of `Local Canonical`? (By the way - heavily insisting :smile: -, what are the motivations and mid-term plans of SSReflect developers w.r.t. this redundancy?)

Reply:
Good question, I wasn't too sure about this one. Do you mean that the current default `locality` for definitions depend on their `kind`? I'll double check.